src/posthog-core.ts

```diff
@@ -1285,13 +1285,13 @@ export class PostHog {
     }
 
     /** Get list of all surveys. */
-    getSurveys(callback: SurveyCallback, forceReload = false): void {
-        this.surveys.getSurveys(callback, forceReload)
+    getSurveys(callback: SurveyCallback): void {
```

Comment:
Removing `forceReload` from public APIs will break customers that are [using the methods directly](https://posthog.com/docs/surveys/implementing-custom-surveys#fetching-surveys-manually).
Is there a reason to remove the `forceReload`?

It'll contract whats written here:

> Surveys are requested on first load and then cached by default by the JavaScript SDK. If you want to force an API call to get an updated list of surveys, pass true to the forceReload parameter. You only need to do this if you want changed surveys to appear mid-session for users.

Reply:
Ewww - I wish we hadn't allowed people to do that tbh. But I can add it back in. We have a bug currently that causes multiple loads anyways